<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPlayer</class>
 <widget class="QMainWindow" name="QPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>文件...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>路径：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_speed_up">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>倍速+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_forward">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>前进10秒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_backward">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>后退10秒</string>
    </property>
   </widget>
   <widget class="QSlider" name="hSlider">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_speed_down">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>倍速-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>312</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
